### ISSUE TITLE
Lock Screen not focusing

### ISSUE THREAD

**Issue by notTanveer** [basecamp/omarchy#7467]
### System details

hp pavilion, amd ryzen 5, gtx 1660

### What's wrong?

After the system automatically goes to the lock screen following the timeout, the password input field doesn’t receive keyboard focus. I have to connect an external mouse and click somewhere on the screen before I can type the password.

Simply typing on the keyboard doesn’t work until the screen is focused with the mouse. This didn’t happen before, and I only started experiencing it after updating Omarchy to version 4.0.0.
